site/en/docs/extensions/whatsnew/index.md

```diff
@@ -15,7 +15,13 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
-### Chrome 116: More than we can fit here {: #chrome-116 }
+### Chrome 115: DevTools 
+
+<p class="color-secondary-text type--caption">Posted on <time>July 24, 2023</time></p>
+
+Injected content scripts are now in the DevTools ignore list by default. This doesn't affect breakpoints, but it does mean that content scripts will be jumped over during debugging exceptions will be ignored. When a content script is open in the **Sources** tab, a banner alerts you that if this is on and provides an option to remove your content script from the ignore list. To turn this behavior off, open DevTools, go to **Settings** then **Ignore list**.
```

Comment:
"exceptions will be ignored" does not fit here.

Would be great to include a screenshot (e.g. the one oliver shared)

Reply:
```suggestion
Injected content scripts are now in the DevTools ignore list by default. This doesn't affect breakpoints, but it does mean that content scripts will be jumped over during debugging and exceptions from these scripts will be ignored. When a content script is open in the **Sources** tab, a banner alerts you if this is on and provides an option to remove your content script from the ignore list. To turn this behavior off, open DevTools, go to **Settings** then **Ignore list**.
```

Reply:
@oliverdunk 
GH won't let me commit your suggestion. It will be in my next push from local. It's also been a few days. Do you remember the screen shot he's talking about?

Reply:
I think the one here: https://developer.chrome.com/blog/new-in-devtools-115/#content-script

Reply:
Actually, I think it's fine without it. We have a link to the DevTools post so that we don't need to provide everything here.